tests/test.py

```diff
@@ -4469,9 +4470,14 @@ def check_all_decryptions(self, **context):
 if __name__ == '__main__':
     import xmlrunner
 
-    output_path = os.environ.get('TEST_XMLOUTPUT', '')
-    if output_path:
-        with open(output_path, 'wb') as output:
-            unittest.main(testRunner=xmlrunner.XMLTestRunner(output=output))
-    else:
-        unittest.main()
+    baseSetUpModule()
+    try:
+        output_path = os.environ.get('TEST_XMLOUTPUT', '')
+        if output_path:
+            with open(output_path, 'wb') as output:
+                unittest.main(testRunner=xmlrunner.XMLTestRunner(output=output))
+        else:
+            unittest.main()
+    finally:
+        baseTearDownModule()
+        sys.exit(0)
```

Comment:
i'm not sure that we should return exit status 0 in the `finally` statement because `finally` executed even if raised some error and we will not know about it and could not handle it in the caller script.